### DISCUSSION TITLE
Microsoft Azure Student Pack

### DISCUSSION THREAD

**Discussion by Patelaarya54** [community/community#39835] [GitHub Education]
![Screenshot_2022-11-24_16-13-11](https://user-images.githubusercontent.com/118821383/203764833-45a2da39-2832-4a33-9154-896c6bd149c3.png)
### Select Topic Area

Bug

### Body

I have GitHub Student Developer Pack and Through GitHub Account I have signed in Into Microsoft Azure and Still cannot get 100$ in my Azure account  Please Do Something @AzureSupport @GitHubHelp

**Comment by david-engelmann** [comment]
[It seems you aren't alone, check out this tread to figure out how to resolve, particularly the last comment](https://learn.microsoft.com/en-us/answers/questions/100252/azure-from-github-student-developer-pack.html)!

**Reply by Patelaarya54** [reply]
This methord is not working properly

**Comment by DSDmark** [comment]
Hi @Patelaarya54,

Make sure, you are using **Gmail** ID. which you use in **GitHub** account then try it again:- [**azure**](https://azure.microsoft.com/en-us/free/students/?WT.mc_id=academic-9938-cxa)

if, in any case, its not working then as it's said in the above image you are not eligible for that, read this:-

> [Azure](https://www.sevenmentor.com/azure-training-institute-in-pune.php) for Students is available only to students who meet the following requirements. You must affirm that you are age 18 or older and attend an accredited, degree-granting two-year or four-year educational institution where you’re a full-time student in a STEM-related field. You must verify your academic status through your organisation’s email address . This offer is not available for use in a massive open online course (MOOC) or in other professional trainings from for-profit organisations. The Azure for Students offer is limited to one per eligible student, is non-transferable and cannot be combined with any other offers unless otherwise permitted by Microsoft. Azure for Students has a limited supply of activations and the offer is available only while supplies last. Each region has a limited number of Azure for Students offers available and your geographical location may run out of supplies while the offer is still available in other regions. Microsoft may terminate this offer at our sole discretion. Other restrictions may apply.


The last things are that. If you are not satisfied: https://azure.microsoft.com/en-us/support/

**Comment by HAMZA10005** [comment]
PLEASE i have the same issue. did you fix it .?
